config/application.rb

```diff
@@ -32,5 +32,7 @@ class Application < Rails::Application
     config.generators do |g|
       g.test_framework :rspec
     end
+
+    config.autoload_paths += %W(app/lib)
```

Comment:
I believe everything in `app` is automatically loaded, which is why we used `app/lib` instead of just `lib`.

Maybe this was your issue? https://stackoverflow.com/a/41371440